src/hooks/productData/posthog_ai.tsx

```diff
@@ -87,22 +88,22 @@ export const posthog_ai = {
             ],
             skills: [
                 {
-                    name: 'Traffic watcher',
-                    description: 'Illuminates pathways through clickthrough chaos and millions of pageviews',
+                    name: 'AI-generated insights',
+                    description: 'AI-generated insights for both event and session-level metrics',
                     sticker: <StickerPath className="size-12" />,
-                    percent: 80,
+                    percent: 30,
                 },
                 {
-                    name: 'UTM untangler',
-                    description: "Makes sense of your team's chaotic campaign naming convention",
+                    name: 'Natural language filters',
+                    description: 'Tell PostHog AI how you want to filter your data, and they shall be applied',
```

Comment:
Maybe something more along the lines of: _Filter by page, user location, browser, etc_ (replace with the most common use cases)